docs/README.md

```diff
@@ -69,7 +69,7 @@ npm install deck.gl
 pip install pydeck
 ```
 
-- [Get started](https://pydeck.gl/installation.html)
+- [Get started](https://deckgl.readthedocs.io/en/latest/installation.html)
 - [Examples](https://pydeck.gl/)
```

Comment:
Probably best if we're consistent about the domain – I don't have a preference on using the pydeck domain or not though. @ibgreen is this OK?


```suggestion
- [Get started](https://pydeck.gl/en/latest/installation.html)
- [Examples](https://pydeck.gl/)
```